.gitmodules

```diff
@@ -1,4 +1,3 @@
 [submodule "cimgui"]
 	path = cimgui
-	url = https://github.com/extrawurst/cimgui
-	branch = docking_inter
```

Comment:
@feliwir is there a reason why are we removing this line?

Reply:
> @feliwir is there a reason why are we removing this line?

The repository migrated to https://github.com/cimgui/cimgui . The only reason this worked was GitHubs auto-redirect. Will revert the branch change though. Thanks!